cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -141,9 +139,11 @@ func createStaticPodFiles(manifestDir string, cfg *kubeadmapi.MasterConfiguratio
 // getAPIServerCommand builds the right API server command from the given config object and version
 func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 	defaultArguments := map[string]string{
-		"advertise-address":               cfg.API.AdvertiseAddress,
-		"insecure-port":                   "0",
-		"admission-control":               defaultAdmissionControl,
+		"advertise-address": cfg.API.AdvertiseAddress,
+		"insecure-port":     "0",
+		// https://github.com/kubernetes/kubernetes/pull/58123
+		"enable-admission-plugin":         "NodeRestriction",
+		"disable-admission-plugin":        "PersistentVolumeLabel",
```

Comment:
Add a TODO to remove this in kubeadm v1.11, as it's automatically disabled in v1.11, and reference https://github.com/kubernetes/kubernetes/pull/64326.
We can't skip it now as we support v1.10 clusters still.